app/assets/stylesheets/comments.scss

```diff
@@ -419,7 +420,7 @@ a.header-link {
       textarea {
         color: lighten($dark-gray, 25%);
         border: 1px solid $light-green;
-        background: white url(image_path('loading-ellipsis.svg')) no-repeat
+        background: white url(image-src('loading-ellipsis.svg')) no-repeat
```

Comment:
https://github.com/thepracticaldev/dev.to/pull/6754/files#r395955475

it seems like yes..